src/Question/Question.js

```diff
@@ -6,15 +6,15 @@ var decode = require('unescape')
 const Question = (props) => {
   if (props.question) {
     let fixedQuestion = props.question
-    if (props.question.includes('&quot;')) {
-      fixedQuestion = props.question.replace(/&quot;/gi, '"')
-    }
-    if (props.question.includes('&#039;')) {
-      fixedQuestion = props.question.replace(/&#039;/gi, "'")
-    }
+    // if (props.question.includes('&quot;')) {
+    //   fixedQuestion = props.question.replace(/&quot;/gi, '"')
+    // }
+    // if (props.question.includes('&#039;')) {
+    //   fixedQuestion = props.question.replace(/&#039;/gi, "'")
+    // }
```

Comment:
I hope we can figure this out :(

Reply:
I have that ridiculous function we can plug in if need be...but that may not work either!